src/core/admin/admin_site.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any
+
+from django.contrib import admin
+
+class AdminSite(admin.AdminSite):
+
+    def __init__(self, *args, **kwargs):
+        super(AdminSite, self).__init__(*args, **kwargs)
```

Comment:
Это конструкция из второго питона, мы пишем на третьем
```suggestion
        super().__init__(*args, **kwargs)
```